### ISSUE TITLE
warn about invalid test module names

### ISSUE THREAD

**Issue by cztomczak** [pytest-dev/pytest#1426]
PyTest throws import errors when there are dots in my files:

```
c:\github\pycef\unittests>py.test
============================= test session starts =============================
platform win32 -- Python 2.7.11, pytest-2.9.0, py-1.4.31, pluggy-0.3.1
rootdir: c:\github\pycef\unittests, inifile: pytest.ini
collected 0 items / 8 errors

=================================== ERRORS ====================================
______________________ ERROR collecting app.Settings.py _______________________
c:\python27\lib\site-packages\py\_path\local.py:650: in pyimport
    __import__(modname)
E   ImportError: No module named app.Settings
______________________ ERROR collecting app.Switches.py _______________________
c:\python27\lib\site-packages\py\_path\local.py:650: in pyimport
    __import__(modname)
E   ImportError: No module named app.Switches
_____________________ ERROR collecting browser.Browser.py _____________________
c:\python27\lib\site-packages\py\_path\local.py:650: in pyimport
    __import__(modname)
E   ImportError: No module named browser.Browser
_________________ ERROR collecting browser.BrowserSettings.py _________________
c:\python27\lib\site-packages\py\_path\local.py:650: in pyimport
    __import__(modname)
E   ImportError: No module named browser.BrowserSettings
____________________ ERROR collecting browser.Embedder.py _____________________
c:\python27\lib\site-packages\py\_path\local.py:650: in pyimport
    __import__(modname)
E   ImportError: No module named browser.Embedder
```

Scripts run fine when executed from command line:
```
python app.Settings.py
```

 It is only PyTest having problems running them.
 
Python 2.7 32-bit.
OS: Win7 Pro SP1

**Comment by The-Compiler** [comment]
Note you won't be able to *import* that file (i.e. using `import app.Settings` in Python) either, so I'm not sure if pytest can (and/or should) support this.

**Comment by RonnyPfannschmidt** [comment]
this is not a bug, tests are imported, and dots are not valid in importable module names

we should warn about that however

**Comment by lgpage** [comment]
Just be careful to allow for [PEP 3149](https://www.python.org/dev/peps/pep-3149/)

**Comment by RonnyPfannschmidt** [comment]
thanks for the warning, i don't think that pep affects us - since the abi tags are not part of a module name

**Comment by hartym** [comment]
Looks resolved by PR #1520 and merged in `features` branch (commit 5685589), works for me.

**Comment by RonnyPfannschmidt** [comment]
thanks for the note, closing as solved via #1520 in commit 5685589
